app/controllers/motorcycles_controller.rb

```diff
@@ -0,0 +1,85 @@
+class MotorcyclesController < ApplicationController
+  before_action :set_motorcycle, only: %i[show edit update destroy]
+
+  # GET /motorcycles or /motorcycles.json
+  def index
+    @motorcycles = Motorcycle.all.order('production_date DESC')
+    # render json: @cities.map { |city| CitySerializer.new(city).serializable_hash[:data][:attributes] }
```

Comment:
- [ ] Kindly, remove this old commented code so your repo's cleanliness is improved.